src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
Maybe I should clarify what I am doing with the kml_xlimits and kml_ylimits figure attributes.  In frametools,  if plotfigure.use_for_kml=True, any plotaxes limits (auto,None, or explicitly set) are overridden by  kml_xlimits/ylimits.   This enforces consistency between the figure axes and the latlong box (created in plotpages and also specified by kml_xlimits/ylimits)  that will be used by GE.

If use_for_kml=False, then the user defined plotaxes limits are used in the usual way.  

I could, for example, not override the plotaxes.xlimits/ylimits settings and just go with whatever the user sets (auto, None, etc).   And use the kml xlimits/ylimits only in creating the kml files in plotpages (and possibly change the name of the attribute).     But then if Matplotlib happens to choose a different set of axes than what is specified in the kml settings, the user would see weird distortions in their plots on the GE browser.   

When creating the KML files (which needs a specified Lat/long box), I could guess at what the user had in mind for axes, but then the other issues of multiple sets of axes, and where to figure out what axes to use comes up.